packages/react-core/src/demos/examples/Skeleton/SkeletonCard.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Gallery, Flex, PageSection, Content, Card, CardBody, Skeleton } from '@patternfly/react-core';
+import { DashboardWrapper } from '@patternfly/react-core/src/demos/DashboardWrapper';
+
+export const SkeletonCard: React.FunctionComponent = () => {
+  const card = (index: number) => (
+    <Card key={index} isCompact>
+      <CardBody>
+        <Flex direction={{ default: 'column' }} spacer={{ default: 'spacerMd' }}>
+          <Skeleton />
```

Comment:
We should pass the `screenreaderText` prop to one of the skeletons in this demo:

```suggestion
          <Skeleton screenreaderText="Loading content" />
```